pkg/internal/testing/controlplane/etcd.go

```diff
@@ -55,48 +57,61 @@ type Etcd struct {
 	Out io.Writer
 	Err io.Writer
 
-	processState *internal.ProcessState
+	// processState contains the actual details about this running process
+	processState *process.State
 }
 
 // Start starts the etcd, waits for it to come up, and returns an error, if one
 // occoured.
 func (e *Etcd) Start() error {
 	if e.processState == nil {
-		if err := e.setProcessState(); err != nil {
+		if err := e.setState(); err != nil {
 			return err
 		}
 	}
```

Comment:
Is there any reason why we need to protect this?

Could we put everything inside a sync.Once?

Reply:
ditto here -- focusing on not changing functionality in this PR